styles/fonts.css

```diff
@@ -70,3 +70,44 @@
     font-weight: 700;
     font-style: italic;
 }
+
+@font-face {
+    font-family: Archivo;
+    src: url('/styles/fonts/Archivo/ArchivoBlack-Regular.ttf') format('truetype');
+    font-display: swap;
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "Chivo Black";
+    src: url('/styles/fonts/Chivo/Chivo-Black.ttf') format('truetype');
+    font-display: swap;
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "Chivo Bold";
+    src: url('/styles/fonts/Chivo/Chivo-Bold.ttf') format('truetype');
+    font-display: swap;
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "Work Sans";
+    src: url('/styles/fonts/Work/WorkSans-Black.ttf') format('truetype');
+    font-display: swap;
+    font-weight: 400;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "Work Sans";
+    src: url('/styles/fonts/Work/WorkSans-Bold.ttf') format('truetype');
+    font-display: swap;
+    font-weight: 700;
+    font-style: normal;
+}
+
```

Comment:
I count 5 additional font-faces but a lot more ttf font assets ? Also please don't use the ttf fonts, instead use the woff2 fonts as they are way more performant.